infra/resources/monitoring.tf

```diff
@@ -1,4 +1,9 @@
-data "azurerm_application_insights" "application_insights" {
-  name                = format("%s-%s-fn", local.project, var.application_basename)
+# Application insights
+resource "azurerm_application_insights" "application_insights" {
+  name                = format("%s-%s-appinsights", local.project, var.application_basename)
+  location            = azurerm_resource_group.rg.location
   resource_group_name = azurerm_resource_group.rg.name
+  application_type    = "Node.JS"
+
+  tags = var.tags
```

Comment:
This will create a new instance of application insights. Is this what we want? I guess we want to collect all logs in the common platform instance, don't we?

Reply:
....actually I don't know!! 😆 I mean, I thought to put them into the common platform instance, but I didn't sure it was right!
If is correct to collect all logs into the common platform instance, I have to change `resource` into `data` and discriminate dev from prod environment in order to get the correct app insight instance 😉

Reply:
I see no problem in using the common instance. I see this app is meant to live short, won't mess monitoring in the long run

Reply:
fixed with 9ea042199dfdb4fd57a46459a91944a057da91a0